.github/workflows/check-ci.yml

```diff
@@ -85,11 +85,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Set up Go
-        uses: actions/setup-go@v4
-        with:
-          go-version: ${{ env.GO_VERSION }}
-
```

Comment:
nice, ty for the cleanup

Reply:
Yeah .. I figured it was not needed anymore.